runner/runners/invoke.go

```diff
@@ -154,7 +155,7 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 	case st = <-processCh:
 	}
 
-	log.Infof("run done. id: %v, status: %+v, cmd: %+v, checkout: %v", id, st, cmd, co.Path())
+	log.Infof("run done. id: %v, status: %+v, checkout: %v", id, st, co.Path())
```

Comment:
I'm punting on this, it can be done as part of a more concerted effort later on if necessary.